.github/workflows/labeler.yml

```diff
@@ -3,6 +3,8 @@ name: Issue Opened Triage
 on:
   issues:
     types: [opened]
+  schedule:
+        - cron: '0 7 * * *' # UTC 7AM (-7)-> 12AM PST
```

Comment:
I believe this is the job that adds the `crash` label, and for service labels we would instead want to change `.github/workflows/backfill-issue-labels.yml`. Schedule config LGTM though.

Reply:
oops, you're right! I'll update it. Thanks for catching it